quic/s2n-quic-core/src/recovery/bandwidth/estimator.rs

```diff
@@ -0,0 +1,173 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::time::Timestamp;
+use core::{cmp::max, time::Duration};
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+/// Bandwidth-related data tracked for each sent packet
+pub struct PacketInfo {
+    /// `Estimator.delivered_bytes` at the time this packet was sent.
```

Comment:
for better linkability in docs and plus you get doc warnings

```suggestion
    /// [`Estimator::delivered_bytes`] at the time this packet was sent.
```

Reply:
For the other fields I get this warning: 
 > public documentation for `delivered_time` links to private item `Estimator::delivered_time`